cpp/README.md

```diff
@@ -86,9 +86,18 @@ following dependencies:
 If your project does not need `lz4` or `zstd` support, you can optionally disable these by defining
 `MCAP_COMPRESSION_NO_LZ4` or `MCAP_COMPRESSION_NO_ZSTD` respectively.
 
+### Conan
+
 To simplify installation of dependencies, the [Conan](https://conan.io/) package
 manager can be used with the included
 [conanfile.py](https://github.com/foxglove/mcap/blob/main/cpp/mcap/conanfile.py).
+
+### CMake
+
+For how to use MCAP with CMake see the [cmake](website/docs/guides/cpp/cmake.md) page.
```

Comment:
I think this link might be broken? In any case, we can probably just link to your repo again here because the internal page doesn't add much info.

```suggestion
For using MCAP with CMake, the third-party [olympus-robotics/mcap_builder](https://github.com/olympus-robotics/mcap_builder) repository provides a helpful wrapper.
```

(Sorry to keep adding feedback as PR suggestions for you to commit - I would have made the changes directly but github doesn't allow me to push to your branch.)

Reply:
no worry, still very easy to commit your suggestion